README.md

```diff
@@ -321,19 +416,31 @@ var config = {
 var authClient = new OktaAuth(config);
 ```
 
-Even if you have specified `localStorage` or `sessionStorage` in your config, the `TokenManager` may fall back to using `cookie` storage on some clients. If your site will always be served over a HTTPS connection, you may want to enable "secure" cookies. This option will prevent cookies from being stored on an HTTP connection.
+A custom [storage provider](#storageprovider) instance can also be passed here. (This will override any `storageProvider` value set under the `token` section of the [storageManager](#storagemanager) configuration)
```

Comment:
old approach override the new one?

Reply:
It is currently operating in compatibility mode. A default set `storageTypes` is set for each well known section. If the config specifies a `storageType` it will be used as the first choice. Then, it will go to the NEXT entry in `storageTypes`. This matches current behavior. Fallback logic can be disabled (see README additions).